docs/codeql/codeql-cli/publishing-and-using-codeql-packs.rst

```diff
@@ -72,3 +72,45 @@ The ``analyze`` command will run the default suite of any specified CodeQL packs
 ::
 
    codeql <database> analyze <scope>/<pack> <scope>/<other-pack>
+
+Managing packs on GitHub Enterprise Server
+------------------------------------------
+
+.. pull-quote::
+
+   Note
+
+   Managing packs on GitHub Enterprise Server is only available for GitHub Enterprise Server 3.6 and later.
+
+By default, CodeQL will download packs from and publish packs to the Container registry on GitHub.com.
+You can manage packs on GitHub Enterprise Server 3.6 and later by creating a ``qlconfig.yml`` file to tell CodeQL which Container registry to use for each pack.
+Create the ``~/.codeql/qlconfig.yml`` file using your preferred text editor, and add entries to specify which registry to use for each pack name pattern.
+For example, the following ``qlconfig.yml`` file associates all packs with the Container registry for the GitHub Enterprise Server at ``GHE_HOSTNAME``, except packs matching ``codeql/*``, which are associated with the Container registry on GitHub.com:
+
+.. code-block:: yaml
+
+   registries:
+   - packages: '*'
+   url: https://containers.GHE_HOSTNAME/v2/
+   - packages: 'codeql/*'
```

Comment:
I'm pretty sure that registries are matched in order, so that means the `*` will match everything before going to `codeql/*`. I think the order needs to be reversed.

And this order dependency needs to be explicitly stated in the docs.